packages/styles/src/mixins/_notification.scss

```diff
@@ -111,7 +111,12 @@
     @include icons-mx.icon($icon);
   }
 
-  // icon override (post-icon)
+  // Prevent flashing when using post-icon
+  &:has(> post-icon)::before {
+    content: none;
+  }
```

Comment:
This fix will not work currently on Firefox, but it's not a big issue and :has will be soon supported.